lms-notify2.php - kombajn do wysyłania powiadomien - web,e-mail i sms :)

jarecky opened the pull request.

chilek commented:
Dasz radę przygotować pull requesta bez tej całej listy zmienionych plików tylko znakiem końca wiersza?

jarecky commented:
Dnia 06/10/15 o godz. 10:42 Tomasz Chiliński napisał(a):

> 
> Dasz radę przygotować pull requesta bez tej całej listy zmienionych plików tylko znakiem końca wiersza?
Chetnie, ale nie wiem jak :)

pozdrawiam
Jarek Dziubek

maciejlew commented:
Prawdopodobnie musisz wycofać tego comita u siebie w prywatnym repo. 
Później w piku .gitattributes wywal wszystkie linie. Po tym git już 
prawdopodobnie nie będzie próbować dodawać tych dodatkowych plików ze 
zmienionymi znakami końca linii. Przygotuj nowego commita (ale bez 
.gitattributes) i prześlij sobie jeszcze raz do repo na gihubie. 
Najlepiej zrób sobie lokalny backup zanim zaczniesz ;)

W dniu 06.10.2015 o 20:24, Jarosław Dziubek pisze:
> Dnia 06/10/15 o godz. 10:42 Tomasz Chiliński napisał(a):
>
> >
> > Dasz radę przygotować pull requesta bez tej całej listy zmienionych 
> plików tylko znakiem końca wiersza?
> Chetnie, ale nie wiem jak :)
>
> pozdrawiam
> Jarek Dziubek
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/lmsgit/lms/pull/407#issuecomment-145954353>.
>

-- 
lion.net.pl - wdrożenia i rozwój Lan Management System

jarecky commented:
Nie działa. Nadal mam:
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   img/tinymce3/utils/mctabs.js
        modified:   img/tinymce3/utils/validate.js
        modified:   lib/xajax/copyright.inc.php
        modified:   lib/xajax/xajax_core/plugin_layer/xajaxEventPlugin.inc.php
        modified:   userpanel/style/default/overlib.js

no changes added to commit (use "git add" and/or "git commit -a")
